packages/components/src/__actions__/Menu/v3/Menu.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import {
+  Menu as RACMenu,
+  MenuProps as RACMenuProps,
+} from "react-aria-components"
+import styles from "./Menu.module.scss"
+
+export type MenuProps = Omit<
+  // @todo: what does it want here instead of any?
+  RACMenuProps<any>,
+  | "selectionMode"
+  | "disallowEmptySelection"
+  | "selectedKeys"
+  | "defaultSelectedKeys"
+>
+
+/**
+ *
+ */
```

Comment:
when we get to documenting this we want to have jsdoc on everything. But to save effort we can just copy/paste the block from RAC